nmos-test.py

```diff
@@ -247,6 +248,10 @@ def index_page():
 
 
 if __name__ == '__main__':
+    if ENABLE_DNS_SD and DNS_SD_MODE == "unicast" and os.geteuid() != 0:

```

Comment:
os.geteuid isn't available on Windows...

Reply:
Hopefully this is sorted now, but I'm yet to try it on a Windows box